docs/getting-started/application/patterns.rst

```diff
@@ -53,122 +53,61 @@ The above policy controls access to the Expense model on the **record level**.
 An actor can only approve an expense if they are the manager of the submitter
 and the expense amount is below a certain limit.
 
-Primary and Secondary Authorization Data
-----------------------------------------
-
-This policy controls access to an ``Expense``, the **primary authorization
-data**.  It relies on other data to make the decision: the submitter of the
-expense (``expense.submitted_by``), and the manager of the submitter
-(``submitted_by.manager``).  This *other* data is called **secondary
-authorization data**.  An important class of secondary authorization data is
-**actor data**.  This data includes properties of the actor, like their role, or
-team membership that is often used in controlling access regardless of whether
-it is over rows, columns or fields.
-
-Where the Policy is Evaluated
------------------------------
```

Comment:
I actually think this content is useful. I don't think we need to make a fuss about defining terms like "policy evaluation point" unless we decide to use them consistently throughout the docs. But I do think it's worth touching on where the policy is evaluated. I think the best thing would be to have some kind of diagram that explains this better. E.g., a diagram showing the flow of the request and where the policy check is happening in this case.